vulkaninfo/CMakeLists.txt

```diff
@@ -114,18 +114,19 @@ if (MINGW)
     target_link_libraries(vulkaninfo PRIVATE ucrt)
 endif ()
 
+if (MSVC)
+    # vulkaninfo ships with the vulkan runtime and needs to meet certain criteria that windows drivers have to pass
```

Comment:
Updated based on info from @charles-lunarg

Reply:
I mean that was more of a guess rather than a hard reason.

Reply:
I can just remove it entirely and add it back in if we realize we need it.

Reply:
That works for me